software/util/buildfiles.py

```diff
@@ -141,33 +156,39 @@ def sitemap(page):
     output.append("</urlset>\n")
     return "".join(output)
 
-def prtocolswap(content,protocol,altprotocol):
-    ret = content.replace("%s://schema.org" % protocol,"%s://schema.org" % altprotocol)
-    for ext in ["attic","auto","bib","health-lifesci","meta","pending"]:
-        ret = ret.replace("%s://%s.schema.org" % (protocol,ext),"%s://%s.schema.org" % (altprotocol,ext))
+
+def protocolSwap(content, protocol, altprotocol):
+    ret = content.replace("%s://schema.org" % protocol, "%s://schema.org" % altprotocol)
```

Comment:
urllib would be a candidate for this.

Reply:
Maybe. I want to add more unit-tests before touching that code.